setup.py

```diff
@@ -1,22 +1,45 @@
+import os
 import setuptools
 
-with open("README.md", "r") as fh:
+with open(os.path.join(os.path.dirname(__file__), "README.md"), "rt") as fh:
     long_description = fh.read()
 
+test_requires = [
+    "pytest",
+    "pytest-cov",
+    "pylint>=1.6.5",
+    "pycodestyle",
+]
+
 setuptools.setup(
-    name="cfg",
-    version="0.0.2",
+    name="diraccfg",
+    use_scm_version=True,
     author="Federico Stagni",
-    # author_email="author@example.com",
     description="DIRAC cfg files reader",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    # url="https://github.com/pypa/sampleproject",
-    packages=setuptools.find_packages(),
+    url="https://github.com/DIRACGrid/cfg",
+    packages=setuptools.find_packages(exclude=["tests"]),
+    license="GPL-3.0-only",
+    test_suite="tests",
+    setup_requires=["setuptools_scm"],
+    install_requires=[],
+    extras_require={
+        'testing': test_requires,
+    },
+    entry_points={
+        'console_scripts': ['diraccfg=diraccfg.__main__:parseArgs'],
+    },
     classifiers=[
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
```

Comment:
Is it correct to declare that 3.5 is supported?

Reply:
I run tests for it